template/src/main/java/com/asyncapi/service/CommandLinePublisher.java

```diff
@@ -1,26 +0,0 @@
-package {{ params['userJavaPackage'] }}.service;
```

Comment:
Add mechanism to disable generation or do not remove

Reply:
Will do